Providers/nxOMSAutomationWorker/automationworker/worker/runtime.py

```diff
@@ -136,7 +136,7 @@ def __init__(self, job_data, runbook):
         self.execution_alias = "python2"
 
         if get_default_python_interpreter_major_version() == 2:
```

Comment:
this method might not be required anymore?

Reply:
Changes are pushed as per discussion